### ISSUE TITLE
[Screen Reader -Microbit menu controls -Variables] Screen reader is not reading proper information of 'make a variable pane' in Scan mode

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2602]
User Experience: 
Users who depends on Screen reader will get confuse if Narrator does not read proper information of 'Make a variable' pane in Scan Mode

Test Environment: 
OS: RS2 Version 1703(OS Build 15063.483) 
Platform: Edge.

Pre-Requisite: Turn on Narrator

Repro Steps:- 
1-Navigate to url:- https://makecode.com/
2-Navigate to the Microbit section element and select Code control given on it.
3-Navigate to the menu controls listed near the Simulator control .
4-Navigate to the Variables control and select it.
5-Change Scan mode by caps+space key
6-Verify if screen reader is reading proper information of make a variable pane or not

Actual Result:- 
In Scan mode(CAPS + Space+ Down/Arrow key), Screen reader is not reading proper information of 'make a variable pane'.

Expected Result:- 
In Scan mode(CAPS + Space+ Down/Arrow key), Screen reader must read proper information of 'make a variable pane'.

Note: 
Same issue exists In Item mode(CAPS + UP/Down Arrow key), 

MAS Reference -
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

Please refer the attachment.
[MAS4.2.1_Variable_NotReadingInfo.zip](https://github.com/Microsoft/pxt/files/1196866/MAS4.2.1_Variable_NotReadingInfo.zip)

**Comment by veler** [comment]
This is part of Blocky, a third-part component.

**Comment by abchatra** [comment]
We have got exception for Blockly bugs from Julia Liuson (CVP). Closing these as Won't Fix. Following is the reasoning.

----------------------

MakeCode can be used in either Text editor or Block editor.

Blocks editor is inherently visual and is based on google blockly. Usage requires drag and drop functionality and we don’t have a equivalence in MakeCode for this w.r.t accessibility. 
We have evaluated couple of research projects which tries to find a equivalence to block based coding for accessibility needs: 
•	Project Torino (Requires students to buy physical blocks)
•	Accessible blockly (This is text based coding without syntax errors)
Both these are drastic departure from the blockly and provides a complete new experience. These are still in research and we intend not to use them in MakeCode at this point of time.

As workaround, the text editor in MakeCode is the way we see students with the need of accessibility using MakeCode.
